compiler/rustc_typeck/src/check/demand.rs

```diff
@@ -360,6 +360,13 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         false
     }
 
+    crate fn hir_id_sole_block_element(&self, hir_id: hir::HirId) -> Option<&'tcx hir::Expr<'tcx>> {
```

Comment:
```suggestion
    crate fn maybe_get_block_expr(&self, hir_id: hir::HirId) -> Option<&'tcx hir::Expr<'tcx>> {
```

Could you add a doc comment too?